package.json

```diff
@@ -16,19 +16,21 @@
     "@fontsource/roboto": "^4.5.8",
     "@mui/material": "^5.10.0",
     "@mui/styled-engine-sc": "^5.10.0",
+    "angular": "^1.8.3",
```

Comment:
is angular needed as a dependency?

Reply:
In my travels of trying to get it working was trying an angular approach. Will remove from dependencies. Good eye.